crates/router/src/connector/signifyd/transformers/api.rs

```diff
@@ -128,19 +186,34 @@ impl TryFrom<&frm_types::FrmSaleRouterData> for SignifydPaymentsSaleRequest {
         };
 
         let created_at = common_utils::date_time::now();
-        let order_channel = OrderChannel::Web;
-        let shipments = Shipments { destination };
+        let order_channel: OrderChannel = OrderChannel::Web;
```

Comment:
Why's this hardcoded to Web?

Reply:
This was hardcoded before, moving this field to payment frm metadata